FakebookNotifications/FakebookNotifications.DataAccess/NotificationsContext.cs

```diff
@@ -37,5 +51,100 @@ public IMongoCollection<Notification> Notifications
+                userCol.DeleteOne(x => x.Email == "testaccount@gmail.com");
+
+                //remove the users notes
+                foreach(var note in davidNotes)
+                {
+                    noteCol.DeleteOne(x => x.Id == note.Id);
+                }
+
+                foreach (var note in testNotes)
+                {
+                    noteCol.DeleteOne(x => x.Id == note.Id);
+                }
+
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        //Seed some initial data
+        public bool SeedData(IMongoCollection<User> userCol, IMongoCollection<Notification> noteCol)
+        {
+
+            try
+            {
+                //Create seed users
+                User user1 = new User();
+                user1.Email = "david.barnes@revature.net";
+                User user2 = new User();
+                user2.Email = "testaccount@gmail.com";
+
+                //insert seed users
+                userCol.InsertOne(user1);
+                userCol.InsertOne(user2);
+
+                //Create Notifications
+                Notification note1 = new Notification()
+                {
+                    Type = new KeyValuePair<string, int>("Post", 1),
+                    LoggedInUserId = "david.barnes@revature.net",
+                    TriggerUserId = "testaccount@gmail.com",
+                    HasBeenRead = false,
+                    Date = DateTime.Now
+                };
+                Notification note2 = new Notification()
+                {
+                    Type = new KeyValuePair<string, int>("Follow", 0),
+                    LoggedInUserId = "david.barnes@revature.net",
+                    TriggerUserId = "testaccount@gmail.com",
+                    HasBeenRead = true,
+                    Date = DateTime.Now
+                };
+                Notification note3 = new Notification()
+                {
+                    Type = new KeyValuePair<string, int>("Like", 15),
+                    LoggedInUserId = "david.barnes@revature.net",
+                    TriggerUserId = "testaccount@gmail.com",
+                    HasBeenRead = false,
+                    Date = DateTime.Now
+                };
+                Notification note4 = new Notification()
+                {
+                    Type = new KeyValuePair<string, int>("Post", 4),
+                    LoggedInUserId = "testaccount@gmail.com",
+                    TriggerUserId = "david.barnes@revature.net",
+                    HasBeenRead = false,
+                    Date = DateTime.Now
+                };
+
+                //Insert Notifications
+                noteCol.InsertOne(note1);
+                noteCol.InsertOne(note2);
+                noteCol.InsertOne(note3);
+                noteCol.InsertOne(note4);
+
+                return true;
+            }
+            catch
+            {
+                return false;
```

Comment:
either we should have logging here, or this catch should be narrowed way down to only the expected exception

Reply:
this is not how we log exceptions. this way (implicitly calling ToString) throws out most of the information about the exception, including the stack trace.. all you need to do is inject an `ILogger<NotificationsContext>`.